api/test/test_events.py

```diff
@@ -104,12 +104,12 @@ def setUp(self):
                 'start': self.event1.start,
                 'end': self.event1.end,
                 'humandate': 'Aug 28-29, 1993',
-                'latitude': 3,
-                'longitude': -2,
+                'latitude': 3.,
+                'longitude': -2.,
                 'venue': 'University',
                 'address': 'On the street',
                 'country': 'US',
-                'url': 'http://url1/',
+                'url': 'https://user.github.io/repository/',
```

Comment:
These are tests for API returning what we want. The tests for `website_url` and `repository_url` are in `workshops.test.test_event.TestEvent.test_website_url`, `workshops.test.test_event.TestEvent.test_repository_url` and `workshops.test.test_event.TestEvent.test_wrong_repository_website_urls`.